proto/plugin.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package hashicorp.waypoint.sdk;
 
-option go_package = ".;proto";
+option go_package = "github.com/hashicorp/waypoint-plugin-sdk;proto";
```

Comment:
No need to change this in this PR. If this is causing real issues, let's open another issue and talk about it. This feels unrelated to the proposed changes here.

Reply:
Fixed